### ISSUE TITLE
Primary Menu Label low contrast

### ISSUE THREAD

**Issue by curtismchale** [proudcity/wp-proudcity#2363]
## Source

- related: https://github.com/proudcity/wp-proudcity/issues/2354
- related: #2362 

## Describe the bug
The label WP provides for the Primary Menu is low contrast. Much like #2362 this is really only there for screenreaders and shouldn't be visible to sighted users in a standard browser. There should be some tag to remove it from so it doesn't get caught in the accessibility check.

Label id `primary-menu-label` to find it.

## QA

Notes:

* 

Links:

*

**Comment by curtismchale** [comment]
Relabelling for https://github.com/proudcity/wp-proudcity/issues/2365 resolved this issue as well.
